concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -1250,38 +1241,56 @@ jobs:
                 export CONCOURSE_URL="https://${CONCOURSE_HOSTNAME}.${SYSTEM_DNS_ZONE_NAME}"
                 export CONCOURSE_ATC_PASSWORD
                 CONCOURSE_ATC_PASSWORD=$(./paas-bootstrap/concourse/scripts/val_from_yaml.rb secrets.concourse_atc_password concourse-secrets/concourse-secrets.yml)
-                export GITHUB_CLIENT_ID
-                export GITHUB_CLIENT_SECRET
 
                 if [ "${ENABLE_GITHUB}" = "true" ] ; then
                   GITHUB_CI_USERS=$(./paas-bootstrap/concourse/scripts/val_from_yaml.rb ci ./paas-bootstrap/concourse/users/users.yml)
                   GITHUB_PROD_USERS=$(./paas-bootstrap/concourse/scripts/val_from_yaml.rb prod ./paas-bootstrap/concourse/users/users.yml)
-                  GITHUB_CLIENT_ID=$(./paas-bootstrap/concourse/scripts/val_from_yaml.rb secrets.github_client_id github-oauth-secrets/github-oauth-secrets.yml)
-                  GITHUB_CLIENT_SECRET=$(./paas-bootstrap/concourse/scripts/val_from_yaml.rb secrets.github_client_secret github-oauth-secrets/github-oauth-secrets.yml)
 
                   if [ "${FLY_TEAM}" = "dev" ] || [ "${FLY_TEAM}" = "ci" ]; then
```

Comment:
Can we use `AWS_ACCOUNT` here instead of `FLY_TEAM` as the latter doesn't really make sense now that this task is only dealing with the "main" team.